system/jlib/jsecrets.cpp

```diff
@@ -1138,15 +1139,32 @@ IPropertyTree *queryTlsSecretInfo(const char *name)
             if (ca)
                 ca->setProp("@path", filepath.str());
         }
-        // TLS TODO: do we want to always require verify, even if no ca ?
-        verify->setPropBool("@enable", true);
+        //For now only the "public" issuer implies client certificates are not required
+        verify->setPropBool("@enable", !strieq(name, "public"));
         verify->setPropBool("@address_match", false);
         verify->setPropBool("@accept_selfsigned", false);
         verify->setProp("trusted_peers", "anyone");
     }
     return info;
 }
 
+IPropertyTree *getIssuerTlsServerConfigWithTrustedPeers(const char *issuer, const char *trusted_peers)
+{
+    IPropertyTree *issuerConfig = queryIssuerTlsServerConfig(issuer);
```

Comment:
If items are ever evicted from the mtlsInfoCache then this would cause problems because it may not be valid any longer.

Reply:
Good point.  Chaged to getIssuerTlsServerConfig() to return a linked instance.